README.md

```diff
@@ -194,483 +234,135 @@ CommandsBefore:
-
-$ iidy describe-stack iidy-demo
-# ... more details ...
-
-$ iidy delete-stack iidy-demo
-# ... confirm Yes ...
-
-
-```
-
-
-### Creating or Executing CloudFormation Changesets
-
-[![asciicast](https://asciinema.org/a/fl9ss0EYyPbDtFmSCaERC2YBU.png)](https://asciinema.org/a/fl9ss0EYyPbDtFmSCaERC2YBU)
-
-### Using Parameter Store for secrets and ARNs
-
-Here's an example of importing a single secret from Parameter Store.
-```YAML
-# example stack-args.yaml
+#### Importing Data
+
+Data can be imported into the stack args file using the `$imports:` block.
+
+| Import Type | Description | Example |
+|-------------|-------------|---------|
+| `file` | Load a local file, JSON and YAML files will be parsed into a data structure | `vars.yaml` |
+| `filehash` | Compute a SHA 256 hash of a file's contents | `filehash:path`, `filehash:?filepath` (`?` prefix allows file to be empty)
+| `filehash-base64` | Compute a SHA 256 hash of a file's contents, returns Base64 encoded hash | `filehash-base64:path`, `filehash-base64:?filepath` (`?` prefix allows file to be empty)
+| `s3` | Fetch an object from S3, JSON and YAML files will be parsed into a data structure | `s3://bucket-name/path/to/file` |
+| `cfn:export` | Fetch a [CloudFormation Export](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/using-cfn-stack-exports.html) | `cfn:export:export-name`, `cfn:export:export-name?region=us-east-1` |
+| `cfn:output` | Fetch the [Output of a CloudFormation stack](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/outputs-section-structure.html) | `cfn:export:stack-name`, `cfn:export:stack-name?region=us-east-1` |
+| `cfn:parameter` | Fetch the value of a parameter of a CloudFormation stack | `cfn:parameter:stack-name/parameter-name`, `cfn:parameter:stack-name/parameter-name?region=us-east-1` |
+| `cfn:tag` | Fetch the value of a tag of a CloudFormation stack | `cfn:tag:stack-name/tag-name`, `cfn:tag:stack-name/tag-name?region=us-east-1` |
+| `cfn:resource` | Fetch a resource of a CloudFormation stack | `cfn:resource:stack-name/resource-name`, `cfn:resource:stack-name/resource-name?region=us-east-1` |
+| `cfn:stack` | Fetch all data of a CloudFormation stack | `cfn:stack:stack-name`, `cfn:stack:stack-name?region=us-east-1` |
+| `http` | Fetch a file over HTTP. JSON and YAML files will be parsed into a data structure | `https://example.com/vars.yaml` |
+| `env` | Fetch value from environment variable | `env:VERSION`, `env:VERSION:default-value` |
+| `git:branch` | Get current git branch name (via `git rev-parse --abbrev-ref HEAD`) | `git:branch` |
+| `git:describe` | Get current git description (via `git describe --dirty --tags`) | `git:describe` |
+| `git:sha` | Get current git sha name (via `git rev-parse HEAD`) | `git:sha` |
+| `random:dashed-name` | Generate a random dashed name, for example: `'uptight-guitar'` | `random:dashed-name` |
+| `random:name` | Generate a random dashed name, for example: `'uptightguitar'` | `random:name` |
```

Comment:
Description for random:name seems to have been copied from name:dashed-name but not changed.